tools/ctl/ctl.go

```diff
@@ -102,25 +102,27 @@ func buildFetchRebuildRequest(ctx context.Context, bench, run, filter string, cl
 }
 
 var tui = &cobra.Command{
-	Use:   "tui [--project <ID>] [--debug-bucket <bucket>] [--benchmark-dir <dir>] [--clean]",
+	Use:   "tui [--project <ID>] [--debug-storage <bucket>] [--benchmark-dir <dir>] [--clean]",
 	Short: "A terminal UI for the OSS-Rebuild debugging tools",
 	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
 		// Exactly one of benchmarkDir or project should be set.
-		if (*benchmarkDir != "") == (*project != "" || *debugBucket != "") {
-			log.Fatal(errors.New("TUI should either be local (--benchmark-dir) or remote (--project, --debug-bucket)"))
+		if (*benchmarkDir != "") == (*project != "" || *debugStorage != "") {
+			log.Fatal(errors.New("TUI should either be local (--benchmark-dir) or remote (--project, --debug-storage)"))
 		}
 		tctx := cmd.Context()
 		var fireClient rundex.Reader
 		if *benchmarkDir != "" {
 			fireClient = rundex.NewLocalClient()
 		} else {
-			if *debugBucket != "" {
-				bucket, prefix, _ := strings.Cut(strings.TrimPrefix(*debugBucket, "gs://"), string(filepath.Separator))
-				if prefix != "" {
-					log.Fatalf("--debug-bucket cannot have additional path elements, found %s", prefix)
+			if *debugStorage != "" {
+				if strings.HasPrefix(*debugStorage, "gs://") {
```

Comment:
maybe parse as a URL instead? looking back at this, that seems like a cleaner solution that prefix matching